src/patternfly/components/AppLauncher/app-launcher.scss

```diff
@@ -0,0 +1,103 @@
+@import "../../patternfly-imports";
+
+.pf-c-app-launcher {
+  // Toggle
+  --pf-c-app-launcher__toggle--PaddingTop: var(--pf-global--spacer--form-element);
+  --pf-c-app-launcher__toggle--PaddingRight: var(--pf-global--spacer--sm);
+  --pf-c-app-launcher__toggle--PaddingBottom: var(--pf-global--spacer--form-element);
+  --pf-c-app-launcher__toggle--PaddingLeft: var(--pf-global--spacer--sm);
+  --pf-c-app-launcher__toggle--MinWidth: var(--pf-global--target-size--MinWidth);
+  --pf-c-app-launcher__toggle--BackgroundColor: transparent;
```

Comment:
won't this be transparent by default?

Reply:
ya this is inherited from the button. I wondered the same. I've seen this in a couple places. I wanted to ask @srambach if she has thoughts on it before I pulled it out of here and the dropdown.